### ISSUE TITLE
Support for IE11

### ISSUE THREAD

**Issue by OriR** [storybooks/addon-jsx#20]
This addon is really awesome! Unfortunately it isn't working on IE11 😞

The cause for this (there might be more) is `react-element-to-jsx-string` is on version `12.0.0` and not `13.0.0`
(There's a long chain of dependencies here but eventually it uses version `2.0.1` of `get-own-enumerable-property-symbols` which now includes an ES5 compiled version of the code).

Not sure if there are any other things that might break IE11, but after disabling just this addon storybook works as expected.

As a workaround - I can take full control over the storybook webpack config and remove the `exclude` from the `.js` rule, that excludes `node_modules`, that way every file will go through babel, but that's highly inefficient and might cause other issues when compiling already compiled modules.

**Comment by wcastand** [comment]
I publish a new version where i upgraded dependencies. let me know if that fix your issue :)

**Comment by OriR** [comment]
Unfortunately I missed some packages a long the way 😞 
I've opened an issue for [`stringify-object`](https://github.com/yeoman/stringify-object/issues/53) and [`react-element-to-jsx-string`](https://github.com/algolia/react-element-to-jsx-string/issues/211)

If it's not to much to ask, I'd like for you to publish a new version once these are resolved 😃

**Comment by wcastand** [comment]
One of them is already closed and the other seems not in their agenda.
Sorry but i don't think i can help you more :/
If they release a version that support browser, i'll publish a new version.

**Comment by ndelangen** [comment]
Is adding https://github.com/SamVerschueren/babel-engine-plugin an option? Or should we do this over at Storybook?

**Comment by brendanmckeown** [comment]
There is a workaround for this issue until `stringify-object` adds support for IE11: add a loader to compile the package with Babel. 

e.g. in your `webpack.config.js` file:
```
  config.module.rules.push({
    test: /\.js/,
    include: path.resolve(__dirname, '../node_modules/stringify-object'),
    use: [{
      loader: 'babel-loader',
      options: {
        presets: ['env']
      }
    }]
  });
```
You may also need to install Babel in your project: 
```
yarn add babel-loader babel-core babel-preset-env
```

**Comment by hipstersmoothie** [comment]
:rocket: Issue was released in v7.1.2 :rocket:
